### DISCUSSION TITLE
zh/guide/%E7%BD%91%E7%BB%9C%E5%9F%BA%E7%A1%80%E8%AE%BE%E6%96%BD/%E6%9C%8D%E5%8A%A1%E5%99%A8%E8%BF%90%E8%A1%8C%E7%8E%AF%E5%A2%83%E6%90%AD%E5%BB%BA

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#748] [Announcements]
# zh/guide/%E7%BD%91%E7%BB%9C%E5%9F%BA%E7%A1%80%E8%AE%BE%E6%96%BD/%E6%9C%8D%E5%8A%A1%E5%99%A8%E8%BF%90%E8%A1%8C%E7%8E%AF%E5%A2%83%E6%90%AD%E5%BB%BA

服务器运行环境搭建 1. 安装 java运行环境 java8 2. 安装MySQL 5.7 3. 安装Redis （服务） 4. 安装Nginx （服务） 5. 安装MySQL客户端 （sqlyog或Navicat） 6. 安装Redis客户端 （annother redis desktop）

http://192.168.1.66:8080/zh/guide/%E7%BD%91%E7%BB%9C%E5%9F%BA%E7%A1%80%E8%AE%BE%E6%96%BD/%E6%9C%8D%E5%8A%A1%E5%99%A8%E8%BF%90%E8%A1%8C%E7%8E%AF%E5%A2%83%E6%90%AD%E5%BB%BA.html

**Comment by chenglinquan** [comment]
哈哈
